domains/measurements/Measurements.API/API/Extensions/IConfigurationExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Serilog;
+using Serilog.Enrichers.Span;
+using Serilog.Formatting.Json;
+
+namespace API.Extensions;
+
+public static class IConfigurationExtensions
```

Comment:
Why this new startup pattern? All our other sub systems are not using it and now we have 3 classes just to start the app. Whats wrong with program.cs? :)

Reply:
Because it is needed to use the TestServerFixture. And that is needed to make the gRPC integration tests. The wallet does the same